libraries/utils/ogl_utils/CMakeLists.txt

```diff
@@ -16,7 +16,16 @@ target_include_directories(ogl_utils PUBLIC
     ${include_dirs}
 )
 
-target_link_libraries(ogl_utils
-    bnb_effect_player
-    glad
-)
\ No newline at end of file
+
+if(DESKTOP_BUILD)
+    target_link_libraries(ogl_utils
+        bnb_effect_player
+        glad
+    )
+elseif(ANDROID_BUILD)
+    target_link_libraries(ogl_utils
+        banuba
+    )
+endif()
+
+
```

Comment:
don't keep empty lines at the end of file, just a newline at the end of the last line is enough